source/installation-guide/packages-list/index.rst

```diff
@@ -58,11 +58,22 @@ Packages List
+| CentOS 7     |  3.6.1  |   64bits    | `wazuh3.6.1_6.4.0 <https://packages.wazuh.com/vm/wazuh3.6.1_6.4.0.ova>`_                     | af61d93a2d115723f23c06744f3d73ad7eda131ba003aef5992f446b394df5adf086083c4d3492209609f58549806387cf3416f298af575e5a86a35348b87a0a | f4c06833b11ae38e4082c08b33b69ff4 |
 +--------------+---------+-------------+----------------------------------------------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------+----------------------------------+
+
+OVF
+---
+
++--------------+---------+-------------+--------------------------------------------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------+----------------------------------+
+| Distribution | Version |Architecture | Package                                                                                    | SHA512 Checksum                                                                                                                  | MD5 Checksum                     |
++==============+=========+=============+============================================================================================+==================================================================================================================================+==================================+
+|              |         |             | `wazuh3.6.1_6.4.0-disk001 <https://packages.wazuh.com/vm/wazuh3.6.1_6.4.0-disk001.vmdk>`_  | aa3e0f03b6d53616fc3e60a88310e9772b7b8017f579b7f1ad2eb98700adb007293a78b4017107ceba8905bd7260b0d60b14fce24cc73f3ae3e86143777ec49b | c0384d10fbfd767e77d4ef3c9f2f1b06 |
++ Solaris 1    |  3.6.1  +   64bits    +---------------------------------------------------------------------------------------------------------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------+----------------------------------+
+|              |         |             | `wazuh3.6.1_6.4.0.ovf <https://packages.wazuh.com/vm/wazuh3.6.1_6.4.0.ovf>`_               | 0845acc7304ff939562530a5e6945d7d9fc6786e4a9f0bed742b66185929fb2190be7bb21c4721936d2dd670ba6e9d2c01b047c9ad1d0aa7a236440fa6793961 | ef32a784c329ce4414a929f596057a58 |
++--------------+---------+--------------+---------------------------------------------------------------------------------------------------------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------+----------------------------------+
```

Comment:
This throws a `WARNING: Malformed table.` error when compiling the table. There are extra table delimiter characters and they're not properly aligned.
![error2](https://user-images.githubusercontent.com/10928321/45407040-93f59780-b668-11e8-9d49-f5cfbad17c74.png)